modules/administration-guide/partials/proc_applying-the-gitlab-authorized-application-secret.adoc

```diff
@@ -0,0 +1,60 @@
+:_content-type: PROCEDURE
+:description: Applying the GitLab-authorized application Secret
+:keywords: applying-the-gitlab-oauth-app-secret, apply-the-gitlab-oauth-app-secret, apply-gitlab-oauth-app-secret, apply-secret, applying-secret, apply-a-secret, applying-a-secret
+:navtitle: Applying the GitLab-authorized application Secret
+// :page-aliases:
+
+[id="applying-the-gitlab-authorized-application-secret_{context}"]
+= Applying the GitLab-authorized application Secret
+
+Prepare and apply the GitLab-authorized application Secret.
+
+.Prerequisites
+* Setting up the GitLab authorized application is completed.
+* The Base64-encoded values, which were generated when setting up the GitLab authorized application, are prepared:
+** *GitLab Application ID*
+** *GitLab Client Secret*
+* `{orch-cli}` is installed in the operating system you are using.
+////
+{orch-cli}=oc
+https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli
+https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/
+////
+
+.Procedure
+
+. Prepare the Secret:
++
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+kind: Secret
+apiVersion: v1
+metadata:
+  name: gitlab-oauth-config
+  namespace: {prod-namespace} <1>
+  labels:
+    app.kubernetes.io/part-of: che.eclipse.org
+    app.kubernetes.io/component: oauth-scm-configuration
+  annotations:
+    che.eclipse.org/oauth-scm-server: gitlab
+    che.eclipse.org/scm-server-endpoint: __<gitlab_server_url>__ <2>
+type: Opaque
+data:
+  id: __<Base64_GitLab_Application_ID>__ <3>
+  secret: __<Base64_GitLab_Client_Secret>__ <4>
+----
+<1> The {prod-short} namespace. The default is `{prod-namespace}`.
+<2> The *GitLab server URL*.
+<3> The Base64-encoded *GitLab Application ID*.
+<4> The Base64-encoded *GitLab Client Secret*.
+
+. Apply the Secret by entering it with `{orch-cli}` on a command line:
++
+[source,subs="+quotes,+attributes,+macros"]
```

Comment:
```suggestion
[subs="+quotes,+attributes,+macros"]
```

Reply:
Same thing in all other occurrences of `source` use when it'\s not a code block but an example command line.

Reply:
Aha, it's an attribute: https://docs.asciidoctor.org/asciidoc/latest/verbatim/source-highlighter/#source-language